installer/Subtitle_Edit_installer.iss

```diff
@@ -897,15 +900,16 @@ function InitializeSetup(): Boolean;
 var
   ErrorCode: Integer;
 begin
-  Result := IsDotNetDetected('v4.7.2', 0); //Returns True if .NET Framework version 4.7.2 is installed, or a compatible version such as 4.8
+  // Returns True if .NET Framework version 4.7.2 is installed, or a compatible version such as 4.8
+  Result := IsDotNetDetected('v4.7.2', 0);
   if not Result then
   begin
-    if not WizardSilent() then 
+    if not WizardSilent() then
     begin
       if SuppressibleMsgBox(CustomMessage('msg_AskToDownNET'), mbCriticalError, MB_YESNO or MB_DEFBUTTON1, IDNO) = IDYES then
         ShellExec('open','https://go.microsoft.com/fwlink/?LinkId=2085155','','',SW_SHOWNORMAL,ewNoWait,ErrorCode);
         Result := False;
       end;
     end;
   end;
-end.
```

Comment:
Hi @XhmikosR,

thx for the PR, works fine, except for the change in line 911 which give a compile error now.

Reply:
@niksedk can you try now? I'm having trouble building the project since I'm using Inno Setup 6.x.

Reply:
Works very well now :)